packages/contracts/src/state-dump/make-dump.ts

```diff
@@ -137,6 +137,7 @@ export const makeStateDump = async (cfg: RollupDeployConfig): Promise<any> => {
       'OVM_ETH',
       'OVM_ExecutionManagerWrapper',
       'OVM_GasPriceOracle',
+      'OVM_SequencerFeeWallet',
```

Comment:
I think the name should be something more along the lines of `OVM_SequencerVault` or something more general as other types of income the sequencer makes could end in the same place

Reply:
Sounds good.

Reply:
Well, actually given that this is only going to be a temporary solution and will only be used for fees I think it's better to keep it to something like `SequencerFeeVault`.